test/runify.js

```diff
@@ -355,7 +359,9 @@ hosts.forEach(function (record) {
         expect(values[4]).toBe("false");
         expect(values[5]).toBe("0");
         expect(values[6]).toBe("1");
-        expect(values[7]).toBe("1.2");
+        if (type !== "libjs") {
+          expect(values[7]).toBe("1.2");
```

Comment:
So... what does libjs print?

Reply:
We're not trying to test for spec conformance here, so just change `expect(values[7]).toBe("1.2");` to some other numeric value with a decimal place that LibJS can represent (and above where it's printed)

Reply:
Sure, changed it to `1.3`.